packages/create-atomic/templates/src/components/results-manager/default.html

```diff
@@ -0,0 +1,63 @@
+<style>
+  .field {
+    display: inline-flex;
+    align-items: center;
+  }
+
+  .field-label {
+    font-weight: bold;
+    margin-right: 0.25rem;
+  }
+</style>
+
+<atomic-result-section-visual>
+  <atomic-result-icon class="icon"></atomic-result-icon>
+</atomic-result-section-visual>
+
+<atomic-result-section-title>
+  <atomic-result-link></atomic-result-link>
+</atomic-result-section-title>
+
+<atomic-result-section-excerpt>
+  <!-- <sample-result-component></sample-result-component> -->
```

Comment:
Should this be uncommented ?

Or adding more comment that explain why it is commented out in the first place ?